docs/helm-chart/production-guide.rst

```diff
@@ -295,6 +295,10 @@ By default, the chart will deploy Redis. However, you can use any supported Cele
   data:
     brokerUrl: redis://redis-user:password@redis-host:6379/0
 
+.. note::
+  It is recommended that you bring your own Redis when using it in production. The built-in Redis instance is more suitable (and intended) for experimentation, enabling you to deploy Airflow quickly.
```

Comment:
Our usage of redis is so simple, I'm not sure this is actually the case. My 2c.

Reply:
ok. I think I was (inadvertently) passing my own opinions here in the doc. I am removing this note, and let users decide. The Celery section begins like that anyway. Thanks for pointing this out